src/main/java/frc/robot/RobotContainer.java

```diff
@@ -143,7 +152,21 @@ private void configureBindings() {
       }
       
     }, swerveDrive));
-  }
+
+    fieldCentricButton.onTrue(new InstantCommand((
+      ) -> {
+        swerveDrive.toggleFieldCentric();
+      }, swerveDrive));
```

Comment:
Instead of creating a new instant command here, instead try using the factory command methods in the `swerveDrive subsystem`.

```java
// In SwerveDrive.java
public Command toggleFieldCentric() {
    return runOnce(() -> {
        fieldCentric = !fieldCentric;
        ntIsFieldCentric.setBoolean(fieldCentric);
    });
```

Then here the code would just be 
```java
fieldCentricButton.onTrue(swerveDrive.toggleFieldCentric());
```

Just a lil easier to read, and you can reuse these commands elsewhere

Reply:
Makes sense I'll be sure to add these changes!